extension/content.js

```diff
@@ -33,6 +33,39 @@ function linkifyBranchRefs() {
 	});
 }
 
+function addMinimizeMaximize() {
+	if (!$('#toc .refined-github-btn-group').length && $('#toc .btn-group').length) {
```

Comment:
Instead of wrapping everything in a `if` statement it would be better to invert the condition and do a early return instead.